src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const changeTab = (neededTab) => {
+    if (neededTab.id !== selectedTab.id) {
+      onTabSelected(neededTab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {
+            tabs.map(tab => (
+              <li data-cy="Tab" key={tab.id} className={tab.id === selectedTab.id ? `is-active` : ''}>
+                <a href={`#${tab.id}`} data-cy="TabLink" onClick={() => changeTab(tab)}>{tab.title}</a>
+              </li>
```

Comment:
```suggestion
              <li 
                data-cy="Tab" 
                key={tab.id} 
                className={tab.id === selectedTab.id ? `is-active` : ''}
              >
                <a 
                  href={`#${tab.id}`} 
                  data-cy="TabLink" 
                  onClick={() => changeTab(tab)}
                >
                    {tab.title}
                </a>
              </li>
```

pls pay attention to your code style.
use  classnames library for conditional classes